File: exercicios/kauan/models/lista03/Tarefa.cpp

#include "models/lista03/Tarefa.h"

#include "utils/ValorString.h"

Tarefa::Tarefa(ValorString titulo, ValorString descricao) :titulo(titulo), descricao(descricao), concluida(false){}

void Tarefa::alterarStatus(){
    this->concluida = !this->concluida;
}

File: exercicios/aula-04/vitor/exercicio1/Tarefa.java
package exercicio1;

public class Tarefa {

    String tarefaName;
    String descricao;
    boolean status;

    void o() {
        if (status == true) {
            System.out.println("Concluida");
        } else {
            System.out.println("Pendente");
        }
    }
}
